packages/runtime/src/runtimes/react/index.tsx

```diff
@@ -79,61 +79,20 @@ export const ReactRuntime = createReactRuntime(storeContextDefaultValue)
 
 registerBuiltinComponents(ReactRuntime)
 
-const Context = createContext(storeContextDefaultValue)
+export const StoreContext = createContext(storeContextDefaultValue)
```

Comment:
This export being shared by the two other modules that we're attempting to code-split might mess with how webpack builds chunks

Reply:
I tested this using the bundle analyzer, and the `PreviewProvider` is only loaded for the Preview page, and not for Live page.

The `ReactPage` is loaded for both Preview and Live page. This is also how it behaves previously.